src/main/java/com/psygate/dedication/listeners/PvPListener.java

```diff
@@ -120,6 +120,9 @@ private void engageMsg(Player attacker, Player victim) {
     }
 
     private void process(EntityDamageByEntityEvent event, Player attacker, Player victim, boolean attackerdicated, boolean victimdedicated) {
+        if (attacker.getUniqueId().equals(victim.getUniqueId())) {

```

Comment:
Yeah was just thinking it's likely overloaded to include Entity cases where you're damaging or being damaged by mobs, I dunno. Who can tell, no documentation ...